src/ui/templates/results/resultsheader.hbs

```diff
@@ -65,7 +62,7 @@
         {{#if _config.changeFiltersText}}
           {{_config.changeFiltersText}}
         {{else}}
-          change filters
+          {{translate phrase='change filters' }}
```

Comment:
does this one need context too? I feel like change filters could be interpreted as something like "change water filter" but I also felt that way for a number of the translations in the yext-js.po files in smartling.

Reply:
Lol totally possible! Updated to added context: "Change is a verb, filters are a set of fixed values or criteria that limit results'"

Reply:
ahh amazing!